### ISSUE TITLE
Slider A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7115]
Tracking all of the issues brought up with the 11/13 accessibility pass on Slider  control
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B4b6c9901-361f-45a3-8f3d-8a1a21062922%7D&action=default&uid=%7B4B6C9901-361F-45A3-8F3D-8A1A21062922%7D&ListItemId=1548&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

- [x] ~~Bug 1: [Accessibility][Screen reader]: In Scan mode user is not able to change the slider value.~~ This is an external issue (Edge team is tracking it internally.)
- [x] ~~Bug 2: [Accessibility][Usable]: Narrator does not narrate the complete information while changing the slider values.~~ (Duplicated by #7993)
- [x] ~~Bug 3: [Accessibility][Screen Reader]: In scan mode narrator does not narrate the slider value.~~ Bugs 3, 4, 5: same behavior on [reference example](https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-1.html) - looks like a byproduct of Bug 1 which is inability to change the slider in scan mode. Seems like the problem's with Edge.
- [x] ~~Bug 4: [Accessibility][Usable]: Scan mode focus is not visible on slider in backward navigation.~~
- [x] ~~Bug 5: [Accessibility][Usable]: Screen does not scroll down along with keyboard focus in back navigation.~~
@micahgodbolt

**Comment by aneeshack4** [comment]
@TestAccessibilityGIT These are all non-repro, please see video below. Perhaps you didn't use Alt + back/forward arrow keys to change slider values properly and use shift + tab for backward navigation - that might be why these didn't work for you.

**Comment by JasonGore** [comment]
Just a general note here: We should be reporting the browser and browser version used when both reporting and responding to issues as no repro.

**Comment by JasonGore** [comment]
@aneeshack4 Microsoft Edge has an active and open issue against #3 (I've also seen this happen with Scan Mode OFF as reported in #7993.) Can you confirm that you tested with Microsoft Edge? I will reopen this issue for now. (If #3 is the only reproed issue we can close and resolve as external.)

**Comment by aneeshack4** [comment]
I just rewatched the screen capture video I made: https://microsoft-my.sharepoint-df.com/:u:/p/aneeshak/EZahg4_gVehHu48vES_mC6UBjD9ZjugJR2Yb6PN5Mc1p9w?e=g3XHlO and it looks like it was in Chrome so I don't think I tested this with Edge.

**Comment by aneeshack4** [comment]
@TestAccessibilityGIT There are no details for Bug 5 in the spreadsheet - can you send a video/further details? (and please respond directly on github, not outlook because the Outlook comments make the issue unnecessarily long and hard to follow.)

**Comment by aneeshack4** [comment]
@TestAccessibilityGIT I'm not able to repro Bug 4 either - when I use the arrows to reverse navigation, focus is not lost. (And this is all on Edge.) I have been trying make a video showing this non repro but Expression Encoder is no longer working on my machine. It's been very frustrating - if you know of any other free screen capture software that I can use for Narrator scenarios, please let me know.

**Comment by awalia13** [comment]
@aneeshack4 - Regarding Bug#5: This is the common issue we found across another control and we mentioned the column "Corresponding Bug ID" in the Bug 5 itself. So please refer the video for that bug.

**Comment by awalia13** [comment]
@aneeshack4 - Regarding Bug#4:  We are still able to reproduce the bug and we tested with below platform and Version:

- Microsoft Edge 44.18836.1000.0
- Microsoft EdgeHTML 18.18836 
- OS Ver: 1903 (OS Build 18836.1000)

**Comment by micahgodbolt** [comment]
seems author has replied but bot didn't remove tag

**Comment by v-doku** [comment]
@JasonGore Could You please provide the Edge bug link which is raised for Bug 1 and 2 for our tracking.

**Comment by JasonGore** [comment]
Please register and link your Github accounts with Microsoft organization so that we can share internal links and bug IDs with you.

**Comment by aneeshack4** [comment]
Tested the remaining 3 on Edge and yes they do repro but the [reference example](https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-1.html
) has the same behavior in Narrator Scan Mode and it's likely a byproduct of Bug 1 which is about Narrator Scan Mode not allowing slider value to change.

**Comment by v-doku** [comment]
@JasonGore have linked my account to Microsoft organization can you please share the bug link now.

**Comment by aneeshack4** [comment]
Sent the Edge links on teams to @v-doku.
